src/Rings/mpoly-ideals.jl

```diff
@@ -2053,13 +2053,14 @@ small_generating_set(I::MPolyIdeal{<:MPolyDecRingElem}; algorithm::Symbol=:simpl
 # Grassmann Pluecker Ideal
 #
 ################################################################################
+#returns Pluecker ideal in ring with standard grading
 function grassmann_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
   I = convert(MPolyIdeal{QQMPolyRingElem},
               Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension))
-  base = base_ring(I)
+  base,x = grade(base_ring(I))
```

Comment:
since you never use `x` here
```suggestion
  base, _ = grade(base_ring(I))
```